ng-abq.adoc

```diff
@@ -252,12 +259,14 @@ $ zxsuite mobile abq allow
 Allow synchronization for a quarantined device
 
 Syntax:
-    zxsuite mobile ABQ allow {device_id}
+   zxsuite mobile ABQ allow {device_id} [attr1 value1 [attr2 value2...]]
```

Comment:
The extra space was added because without it the content was rendered in an anomalous way in my testing environment, but I can remove it if it renders properly here.